friends/package.json

```diff
@@ -28,13 +28,15 @@
   },
   "dependencies": {
     "minimist": ">=0.2.1",
-    "cloudstate": "0.0.3"
+    "cloudstate": "0.0.3",
+    "@grpc/proto-loader": "^0.1.0",
+    "google-protobuf": "^3.0.0",
+    "grpc": "^1.20.2"
   },
   "scripts": {
-    "prestart": "compile-descriptor friends.proto",
+    "build": "compile-descriptor ./friends.proto",
     "start": "node index.js",
-    "start-no-prestart": "node index.js",
-    "dockerbuild": "docker build -t ${DOCKER_PUBLISH_TO:-cloudstateio}/samples-js-chat-friends:latest .",
+    "dockerbuild": "docker build -f ./Dockerfile -t ${DOCKER_PUBLISH_TO:-cloudstateio}/samples-js-chat-friends:latest ../..",
```

Comment:
`-f Dockerfile` can be replaced by `.`

Reply:
What is the last `../..` for?

Reply:
I'd like to know why too, took it from [samples-js-shoppingcart/shopping-cart/package.json](https://github.com/cloudstateio/samples-js-shoppingcart/blob/master/shopping-cart/package.json#L45)